picdar-export/src/main/scala/com.gu.mediaservice.picdarexport/lib/cleanup/UsageRightsOverride.scala

```diff
@@ -0,0 +1,55 @@
+package com.gu.mediaservice.picdarexport.lib.cleanup
+
+import com.gu.mediaservice.lib.config.{MetadataConfig, PhotographersList}
+import com.gu.mediaservice.model._
+
+object UsageRightsOverride {
+
+  def removeCommissioned(s: String) = s.replace("(commissioned)", "").trim
+  def getPublication(s: String) = PhotographersList.getPublication(MetadataConfig.allPhotographers, s).getOrElse("The Guardian")
```

Comment:
No it's not - this is a larger conversation about what publication is.
For now this is fine.